test/example-prometheusrule-spec.yaml

```diff
@@ -0,0 +1,23 @@
+groups:
+  - name: JSONNET_RULE
+    rules:
+      - alert: TEST_ALERTNodesHealthHighHighHigh2
```

Comment:
This name of example alert is leftover of my testing. It can be `TEST_ALERTNodesHealthHigh`. 
Also in [prom-rule-jsonnet.yaml](https://github.com/dNationCloud/kubernetes-jsonnet-translator/blob/e2e-tests/examples/prom-rule-jsonnet.yaml).